src/java.desktop/macosx/native/libawt_lwawt/awt/AWTView.m

```diff
@@ -965,7 +968,7 @@ - (void) insertText:(id)aString replacementRange:(NSRange)replacementRange
 
     if ((utf16Length > 2) ||
         ((utf8Length > 1) && [self isCodePointInUnicodeBlockNeedingIMEvent:codePoint]) ||
-        ((codePoint == 0x5c) && ([(NSString *)kbdLayout containsString:@"Kotoeri"]))) {
+        [(NSString *)kbdLayout containsString:@KEYMAN_LAYOUT]) {
 #ifdef IM_DEBUG
```

Comment:
As noted in JBS Kotoeri is obsolete, which is why the reference to it here should be removed as part of this change.